src/main/java/com/volmit/iris/core/command/pregen/CommandIrisPregenCreate.java

```diff
@@ -0,0 +1,264 @@
+package com.volmit.iris.core.command.pregen;
+
+import com.volmit.iris.Iris;
+import com.volmit.iris.core.gui.PregeneratorJob;
+import com.volmit.iris.core.pregenerator.PregenTask;
+import com.volmit.iris.core.tools.IrisToolbelt;
+import com.volmit.iris.util.collection.KList;
+import com.volmit.iris.util.math.Position2;
+import com.volmit.iris.util.plugin.MortarCommand;
+import com.volmit.iris.util.plugin.VolmitSender;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+
+import java.util.Arrays;
+
+
+public class CommandIrisPregenCreate extends MortarCommand {
+
+    private static final KList<String> argus = new KList<>("radius=", "width=", "height=", "x=", "z=");
+
+    public CommandIrisPregenCreate() {
```

Comment:
This should probably be "start" instead of "create". Same for the command name

Reply:
You got it